frontend/coprs_frontend/run/check_for_anitya_version_updates.py

```diff
@@ -157,11 +157,15 @@ def is_prerelease(version: str) -> bool:
     """
     Detect a pre-release version string.
```

Comment:
I think the function name `is_prerelease` is fine but can you please update the docstring so it is clear that it filters both pre-release and post-release versions?

Reply:
I was thinking about it, whether "post" isn't actually just another "pre" release :-)

Reply:
Updated.